superset/translations/it/LC_MESSAGES/messages.po

```diff
@@ -2274,16 +2275,16 @@ msgstr ""
 #: superset/assets/javascripts/explore/stores/visTypes.js:7
 #: superset/assets/javascripts/explore/stores/visTypes.js:31
 msgid "Time"
-msgstr ""
+msgstr "Tempo"
 
 #: superset/assets/javascripts/explore/stores/visTypes.js:9
 #: superset/assets/javascripts/explore/stores/visTypes.js:32
 msgid "Time related form attributes"
-msgstr ""
+msgstr "Attributi relativi al tempo"
 
 #: superset/assets/javascripts/explore/stores/visTypes.js:16
 msgid "Datasource & Chart Type"
-msgstr ""
+msgstr "Sorgente dati e Tipo di grafico"
```

Comment:
Lowercase tipo. In other places you haven't translated datasource, i think we should decide to translate or not and be consistent. But we could revise this later.